package.json

```diff
@@ -4,15 +4,13 @@
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ zer0-os/zns",
-    "create-local": "graph create --node http://localhost:8020/ zer0-os/zns",
-    "remove-local": "graph remove --node http://localhost:8020/ zer0-os/zns",
-    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 zer0-os/zns"
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ stan36/ZNS-subgraph",
```

Comment:
Is `stan36` your user account? Could we change this to be something more generic or remove the direct reference to `stan36`

Reply:
stan36 is my user account, im not sure  to be honest.  I didnt create this subgraph but it was transfered to me so maybe this is because of that? I'll investigate changing it.